backend/development.ini.sample

```diff
@@ -498,3 +512,10 @@ level=NOTSET
 # number of max log file = 5
 args=('tracim.log','a',10*1024*1024,5)
 formatter=generic
+
+# file log for tracim_notification
+[handler_tracim_notification]
+class=FileHandler
+level=INFO
+args = ('%(here)s/tracim_notification.log','a')
```

Comment:
we do not use `%(here)` variable for `handler_logfile`, I suggest to use a similar configuration

Reply:
Yes, We can write directly tracim_notification.log but we should be careful with relative path. I've got  some trouble without finding were the log was stored, so i set default to this working value.